automation/src/main/java/org/wso2/carbon/testgrid/automation/executers/TestNgExecuter.java

```diff
@@ -18,23 +18,65 @@
 
 package org.wso2.carbon.testgrid.automation.executers;
 
+import org.testng.TestNG;
 import org.wso2.carbon.testgrid.automation.exceptions.TestGridExecuteException;
 import org.wso2.carbon.testgrid.automation.executers.common.TestExecuter;
 import org.wso2.carbon.testgrid.common.Deployment;
 
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+
 /**
  * This class is responsible for Executing TestNG tests.
  */
 public class TestNgExecuter implements TestExecuter {
```

Comment:
Typo in the name 'Executer'. It should be Executor! :) 
[See proof ;)](https://docs.oracle.com/javase/tutorial/essential/concurrency/executors.html)

Reply:
TThis cannot be fixed in this PR and will be corrected through Issue #41